ci/test/cargo-test/mzcompose.py

```diff
@@ -28,6 +28,7 @@
             "KAFKA_ADVERTISED_LISTENERS=HOST://localhost:30123,PLAINTEXT://kafka:9092",
             "KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=HOST:PLAINTEXT,PLAINTEXT:PLAINTEXT",
         ],
+        auto_create_topics=True
```

Comment:
This doesn't look right, we should fix up the specific tests that need a topic by creating it